src/main/java/world/bentobox/bentobox/managers/IslandsManager.java

```diff
@@ -703,9 +717,12 @@ public void load(){
         islandCache.clear();
         quarantineCache.clear();
         List<Island> toQuarantine = new ArrayList<>();
-        // Only load non-quarantined island
+        // Attempt to load islands
         handler.loadObjects().stream().forEach(island -> {
-            if (island.isDoNotLoad() && island.getWorld() != null && island.getCenter() != null) {
+            if (island.isDeleted()) {
+                // These will be deleted later
```

Comment:
If this occurs in the future, we can then do a purge command or something to clean up the database.